cmd/infra/ibmcloud_powervs/create.go

```diff
@@ -13,37 +24,39 @@ import (
 	"github.com/IBM/platform-services-go-sdk/resourcecontrollerv2"
 	"github.com/IBM/platform-services-go-sdk/resourcemanagerv2"
 	"github.com/IBM/vpc-go-sdk/vpcv1"
-	"github.com/openshift/hypershift/cmd/log"
-	powerUtils "github.com/ppc64le-cloud/powervs-utils"
-	"github.com/spf13/cobra"
-	"os"
-	servicesUtils "sigs.k8s.io/cluster-api-provider-ibmcloud/pkg/cloud/services/utils"
-	"strings"
-	"time"
+)
+
+const (
+	defaultCloudConnSpeed = 5000
+
+	cisService         = "internet-svcs"
+	cisServicePlan     = "standard"
+	powerVsService     = "power-iaas"
+	powerVsServicePlan = "power-virtual-server-group"
+
+	availableState   = "available"
+	activeStateL     = "active"
+	activeStateU     = "ACTIVE"
+	failedStateU     = "FAILED"
+	errorStateU      = "ERROR"
+	failedStateL     = "failed"
+	errorStateL      = "error"
+	establishedState = "established"
```

Comment:
Not very clean for what resource state these are?! it will be great if you can group then and rename the variables to represent the resource type as well like, add more comments.

Reply:
refactored.